builtins/amp-ad.js

```diff
@@ -227,7 +232,7 @@ export function installAd(win) {
       // it is visible.
       if (inViewport) {
         this.unlistenViewportChanges_ =
-            this.getViewport().onChanged(this.sendAdIntersection_.bind(this));
+            this.getViewport().onScroll(this.sendAdIntersection_.bind(this));
```

Comment:
Ok. The only case where I know Safari sends reliably more the one scroll event per frame is overscroll, which I believe we've already ignoring.